openghg_defs/data/site_info.json

```diff
@@ -3096,25 +3096,27 @@
   },
   "WAO": {
     "ICOS": {
-      "height": ["21m"],
-      "height_name": ["20magl"],
-      "height_station_masl": 10.0,
-      "latitude": 52.95042,
-      "long_name": "Weybourne Observatory, UK",
-      "longitude": 1.12194
+      "height": ["10m"],
+      "height_name": ["10magl"],
```

Comment:
@ag12733 - Would you want the "height_name" values for "WAO" kept as "20magl" rather than updated to "10magl"?